From: Pekka Paalanen <pekka.paalanen@collabora.com>
Date: Fri, 22 Mar 2019 13:58:30 +0200
Subject: [PATCH] meson: link editor with gobject-2.0

editor.c calls g_clear_object(), so it should link to gobject directly instead
of relying on pangocairo pulling it in in its pkg-config.

Fixes: https://gitlab.freedesktop.org/wayland/weston/issues/211

Signed-off-by: Pekka Paalanen <pekka.paalanen@collabora.com>
---
 clients/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/clients/meson.build b/clients/meson.build
index 47e9e8ce3d4f..53a94212fbf1 100644
--- a/clients/meson.build
+++ b/clients/meson.build
@@ -236,7 +236,7 @@ demo_clients = [
 			text_input_unstable_v1_client_protocol_h,
 			text_input_unstable_v1_protocol_c,
 		],
-		'deps': [ 'pangocairo' ]
+		'deps': [ 'pangocairo', 'gobject-2.0' ]
 	},
 	{ 'basename': 'eventdemo' },
 	{ 'basename': 'flower' },
